FirebaseAI/Sources/Types/Public/Schema.swift

```diff
@@ -167,19 +167,21 @@ public final class Schema: Sendable {
   /// - Parameters:
   ///   - description: An optional description of what the string should contain or represent; may
   ///     use Markdown format.
+  ///   - title: An optional human-readable name/summary for the object schema.
```

Comment:
nit: Would you mind changing these from "the object schema" to either "the <type> schema" or just "the schema"? I don't feel strongly either way, whichever you think is best.